finetune/adapter.py

```diff
@@ -78,15 +74,14 @@ def main(
     mark_only_adapter_as_trainable(model)
 
     num_params = sum(p.numel() for p in model.parameters() if p.requires_grad)
-    print(f"Number of trainable parameters: {num_params}")
+    fabric.print(f"Number of trainable parameters: {num_params}")
```

Comment:
Just curious what this does compared to regular `print`. In this case, if you are running on 8 GPUs, for example, the print statement would not be duplicated in the output terminal?